monday/exceptions.py

```diff
@@ -1,2 +1,6 @@
 class MondayError(RuntimeError):
     pass
+
+
+class MondayQueryError(Exception):
```

Comment:
[very non-blocking] I don't know why we inherit from `RuntimeError` for `MondayError`, but maybe we should follow that pattern?  I think it's wrong, personally, but I don't have the context.  Alternately, this could inherit from `MondayError`, but that also seems wrong, as it doesn't seem to be used anywhere.  Maybe it can go away?  I don't know.

Reply:
I genuinely do not know or remember. That can almost certainly go away.

Reply:
should definitely inherit from `Exception`

Reply:
Technically, `MondayError` (base exception class for this library) should inherit from `Exception` and `MondayQueryError` should inherit from `MondayError`. This makes it very easy to handle "all" potential exceptions that may be thrown inside the monday client by calling code:

```python
try:
   pass
except MondayError:
   pass
```